gitbanner.js.org

asheeeshh opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)
- [x] I have added a CNAME file to my repo: https://github.com/asheeeshh/gitbanner/blob/gh-pages/CNAME

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But the page is not showing any content because the paths have changed. Could you please fix this to give me the chance to have a look.
I'll merge after we have "contact". Please keep this PR open.

asheeeshh commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But the page is not showing any content because the paths have changed. Could you please fix this to give me the chance to have a look. I'll merge after we have "contact". Please keep this PR open.

I changed the path, and the app works just fine now: https://gitbanner.js.org :)
Thank you!

indus commented:
Welcome https://gitbanner.js.org
